task3/src/models/user.js

```diff
@@ -0,0 +1,39 @@
+import { Model, DataTypes } from 'sequelize';
+import { v4 as uuid } from 'uuid';
+
+export default class User extends Model {
+    static init(sequelize) {
+        return super.init(
+            {
+                id: {
+                    type: DataTypes.UUID,
+                    defaultValue: DataTypes.UUIDV4,
```

Comment:
defaultValue: uuid() or uuid-ossp postgres extension can be used.

Reply:
both **defaultValue: uuid** and **defaultValue: literal('uuid_generate_v4()')** are working, I'd stick to last one as we won't need extra package in that case